### ISSUE TITLE
Add support for saving watermark/state for file based incremental replications

### ISSUE THREAD

**Issue by ShahBinoy** [slingdata-io/sling-cli#134]
Hi,
As per discussion here https://github.com/slingdata-io/sling-cli/discussions/131, this request is to add support for incremental replications to file destinations.

**Comment by ShahBinoy** [comment]
@flarco is there a roadmap planned for this feature ?

**Comment by flarco** [comment]
@ShahBinoy yes it is, worked on it for a bit, but didn't finish. Hopefully in the next few weeks.

**Comment by flarco** [comment]
This has been added in the latest version. Please see details [here](https://docs.slingdata.io/examples/database-to-file/incremental). Closing.

**Comment by flarco** [comment]
hey @mrbungie , that is correct.
